lib/inline_svg/transform_pipeline/transformations.rb

```diff
@@ -24,11 +24,10 @@ def self.custom_transformations
   end
 
   def self.magnify_priorities(transforms)
-    transforms.inject({}) do |output, (name, definition)|
+    transforms.each_with_object({}) do |(name, definition), output|
       priority = definition.fetch(:priority, built_in_transformations.size)
 
       output[name] = definition.merge({ priority: magnify(priority) })
-      output
```

Comment:
Not sure about this one. I know some people don't like that it's necessary to remember to return the `memo` object from the block but I've never loved the explicit mutability of the object in `each_with_object`. 🤔

Reply:
I've disabled this cop with a reference to this comment and force pushed